### PR TITLE
sample_id model modification

### PR DESCRIPTION
**Pull request by NuriaQueralt**
hi @rajaram5 I modified sample_id model following the EJP IBM use case. I have added an RDF/TTL example, a PNG image and a ShEx file. Pls, could you review these model modifications and files? NB that I changed NCIT types to other OBO ontologies types like in IBM. tx!

### CODE REVIEW

File: example-rdf/turtle/sample/sampleId.ttl [hunk 1]
```diff
@@ -1,11 +1,15 @@
-@prefix : <http://purl.org/ejp-rd/phenopackets-rdf-schema/v010/example-rdf/> .
+@prefix : <http://purl.org/ejp-rd/ibm/metadata/example-rdf/> .
```
Comment (rajaram5) [inline example-rdf/turtle/sample/sampleId.ttl:1]: @NuriaQueralt can you use phenopacket base url? Just to keep it genral

**Review by rajaram5 — CHANGES_REQUESTED** [review]
Can you fix the rdf base url?

**Comment by NuriaQueralt** [comment]
Hi @rajaram5 I have fixed the  RDF base URI from the ibm to phenopackets as you requested. Could be finished now the PR? tx!

**Review by rajaram5 — APPROVED** [review]

**Comment by rajaram5** [comment]
Looks good, can you please merger it?

**Pull request merged**